spec.html

```diff
@@ -40816,7 +40952,7 @@ <h1>Regular Expressions</h1>
   <h1>Additional ECMAScript Features for Web Browsers</h1>
   <p>The ECMAScript language syntax and semantics defined in this annex are required when the ECMAScript host is a web browser. The content of this annex is normative but optional if the ECMAScript host is not a web browser.</p>
   <emu-note>
-    <p>This annex describes various legacy features and other characteristics of web browser ECMAScript hosts. All of the language features and behaviours specified in this annex have one or more undesirable characteristics and in the absence of legacy usage would be removed from this specification. However, the usage of these features by large numbers of existing web pages means that web browsers must continue to support them. The specifications in this annex define the requirements for interoperable implementations of these legacy features.</p>
+    <p>This annex describes various Legacy features and other characteristics of web browser ECMAScript hosts. All of the language features and behaviours specified in this annex have one or more undesirable characteristics and in the absence of legacy usage would be removed from this specification. However, the usage of these features by large numbers of existing web pages means that web browsers must continue to support them. The specifications in this annex define the requirements for interoperable implementations of these Legacy features.</p>
```

Comment:
I don't think this should be capital-L legacy, given that we've been saying we need consensus to mark something as Legacy when moving it out of Annex B.

Also I don't think it adds anything to link to the definition of Legacy here. This is just the normal usage of the term.

Reply:
Everything in Annex B is already "Legacy", though, due to the existing wording - the consensus requirement is to allow something to *no longer* be legacy once hoisted.